### ISSUE TITLE
Recalls and alerts: Information about the type of recall or alert is not provided logically

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#163]
**Page**: All recalls and alerts
**URL**: e.g. https://rsams.dev.cloud.openplus.ca/en/alert-recall/transport-canada-recall-2022017-nissan
**User Agent**: Firefox (v99)
**language**: English and French

**Issue Description**: The text provided above the "h1" about what type of recall or alert could be missed by screen reader users if they jump to the main content of the page by activating the "Skip to main content" link at the top of the page. The "Skip to main content" link takes users to the "h1" heading.

![image](https://user-images.githubusercontent.com/12256067/162269840-f6f77d22-fcfa-460a-ba2f-6fd86a96b591.png)

![image](https://user-images.githubusercontent.com/12256067/162270086-1831e2fc-c6d2-48e4-bd9b-958e3f5096b6.png)

**Steps to reproduce:**
1. Go to the above URL
2. Visual scanning

**Expected results**: In this case, that text should be displayed below the "h1" heading so that it will not be missed by the screen reader users if they jump to the main content.

**Comment by echouin** [comment]
This is as intended. Resolved.

**Comment by 40999221** [comment]
Issue is still exist. Information about the type of recall/alert is not below the h1 heading as suggested. As per the previous comment, its as per the requirement and may be thats why no change applied yet.
![image](https://github.com/user-attachments/assets/0c826b9a-5b03-4daa-ab96-7401b43275c9)

**Comment by zimw7** [comment]
Issue still exists. Activating the 'Skip to main content' link jumps to the h1, missing the text above it. 
This text should be below the h1 to ensure screen reader users don't miss it.
![Snap1](https://github.com/user-attachments/assets/68873dfb-d932-40a0-b67e-100fae5156b3)

**Comment by YuxuanXie01** [comment]
Issue persists, but according to a previous comment by echouin, this is the intended behavior.

**Comment by mmiron-HC** [comment]
@40999221 Please copy this ticket over to [ART-2605](https://jill.hc-sc.gc.ca/jira/browse/ART-2605)
